config/webhook.yaml

```diff
@@ -105,6 +105,15 @@ spec:
         # containerPort "https-webhook" to the same value.
         - name: WEBHOOK_PORT
           value: "8443"
+        # if you change WEBHOOK_ADMISSION_CONTROLLER_NAME, you will also need to update
+        # the webhooks.name in 500-webhooks.yaml to include the new names of admission webhooks.
+        # Additionally, you will also need to change the resource names (metadata.name) of
+        # "mutatingwebhookconfigurations" and "validatingwebhookconfigurations" in 500-webhooks.yaml
```

Comment:
Ahh I found this should be in singular form `mutatingwebhookconfiguration` and `validatingwebhookconfiguration` :P

Reply:
I agree @XinruZhang, it ideally fits in the singular form but the `resources` section has all the resources in plural for a reason 🙃 

https://github.com/tektoncd/pipeline/blob/9814fddf5cebc85419794958a1d93ebdcc3b57aa/config/200-clusterrole.yaml#L106

The reason being the `resources` are part of the API and accessed using a string representation of their object name, such as `mutatingwebhookconfigurations` for MutatingWebhookConfiguration, `pods` for a Pod, etc.

Reply:
Ahhh I should've be clearer! Yes yes right it is plural in the 200-clusterrole.yaml. I tried to point out it is singular in 500-webhooks.yaml -- I tried to search for "mutatingwebhookconfigurations" and "validatingwebhookconfigurations" in 500-webhooks.yaml to see what  users need to do but supprisingly failed then figured out they were in singular form :P

Maybe using "MutatingWebhookConfiguration" and "ValidatingWebhookConfiguration"?

Reply:
I see what you mean 👍 

Sorry for the confusion, yeah it should be the resource title `MutatingWebhookConfiguration` and `ValidatingWebhookConfiguration`.

Thanks a bunch @XinruZhang for following through the instructions and for a careful review, appreciate it! 🙏 🤗 

Let me know if you find anything else that I need to change ✍️